superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -166,6 +167,17 @@ export default class MainPreset extends Preset {
         new TimeGrainFilterPlugin().configure({ key: 'filter_timegrain' }),
         new EchartsTreeChartPlugin().configure({ key: 'tree_chart' }),
         new HandlebarsChartPlugin().configure({ key: 'handlebars' }),
+        new SupersetOlPlugin({
+          defaultLayers: [
```

Comment:
maybe we can use a different default layer here that is more generic. for example the standard OSM tile

Reply:
Sure. I will add a TODO